packages/drpc/src/DrpcRole.ts

```diff
@@ -0,0 +1,4 @@
+export enum DrpcRole {
```

Comment:
Can these be relocated under `drpc/src/models`? Slightly more consistent with the rest.